src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -441,6 +447,13 @@ public boolean isOpen() {
     @Override
     public void close() throws IOException {
         if (open) {
+            exec.shutdown();
+            try {
+                exec.awaitTermination(3, TimeUnit.MINUTES);
```

Comment:
Oh, I see, I figured it would kill the thread if the timer ran out.  Should we just aggressively kill the thread here?  I'm assuming if we're closing we want to stop the prefetching immediately.  I think we don't want to close the channel and end up with surprising uncaught IOExceptions.